package.json

```diff
@@ -22,7 +22,7 @@
     "chai": "^3.5.0"
   },
   "scripts": {
-    "start": "node app.js",
+    "start": "node out/app.js",
```

Comment:
I'm wondering if we should add a `tsc` call before that. Using WebStorm/IntelliJ our `.ts` files are compiled automatically on save, but on our Azure VMs we will have to run `tsc` manually.